Vagrantfile

```diff
@@ -88,9 +88,35 @@ Vagrant.configure(2) do |config|
     bash nodesource_setup.sh
     apt-get -y install nodejs
     cd /vagrant
-    echo "npm install"
-    echo "npm run watch"
-#    npm install
-#    npm run watch
+    # node_modules folder breaks when setting up in Windows, so use binding to fix
+    echo "Preparing local node_modules folder…"
+    mkdir -p /vagrant_node_modules
+    mkdir -p /vagrant/node_modules
+    chown vagrant:vagrant /vagrant_node_modules
+    mount --bind /vagrant_node_modules /vagrant/node_modules
+    npm install
+    git clone https://github.com/pouchdb/add-cors-to-couchdb.git
+    cd add-cors-to-couchdb
+    npm install
+    npm install -g @angular/cli
+    git clone https://github.com/ole-vi/planet.git
+    echo "inside planet folder"
+    cd planet
+    echo "install"
+    npm install
+    node bin.js http://localhost:5984
+    cd ..
+    curl -X PUT http://127.0.0.1:5984/_users
+    curl -X PUT http://127.0.0.1:5984/_replicator
+    curl -X PUT http://127.0.0.1:5984/_global_changes
+    sudo -u vagrant screen -dmS build bash -c 'cd /vagrant; ng serve'
   SHELL
-end
+  
+  # Run binding on each startup make sure the mount is available on VM restart
+  config.vm.provision "shell", run: "always", inline: <<-SHELL
+    docker start planet
+    # mount --bind /vagrant_node_modules /vagrant/node_modules
```

Comment:
Why is this line commented out?  This is used to reestablish node_modules folder binding on startup.